.coveragerc

```diff
@@ -11,4 +16,8 @@ exclude_lines =
     raise AssertionError
 
 [run]
+branch = false
```

Comment:
In a separate PR I will set this to `true`. The coverage tool then highlight some branching paths that aren't full explored, even if every line is technically executed.

https://coverage.readthedocs.io/en/6.3.1/branch.html